Header header

obstacle_detector/Point2D[] points
